docs/source/upcoming_release_notes/1363-update-dream-motion-gui.rst

```diff
@@ -0,0 +1,30 @@
+1363 update-dream-motion-gui
+#################
+
+API Breaks
+----------
+- N/A
```

Comment:
It's maybe a little bit disingenuous to delete four classes and say there are no API breaks. When we merge this you need to find all the places where these are loaded in tmo hutch python and tmo guis and and swap them out.

Reply:
I see. Thanks Zach. If this is the case, maybe I want to hold this until summer down time. How can I hold this?

Reply:
Just ask us not to merge if you want to wait
Maybe I'm being a bit hyperbolic: it won't be too hard to track down if DREAM devices are used anywhere already. But we should track them down before merging because we're deleting them.

Reply:
Thanks Zach. I will need to test those and add happi, then will hold until summer down time.

Reply:
Tong, the other thing you could consider is Robert's suggestion:
> make an additional device that combines the components and leave the old ones alone

So, if you leave all the old devices alone, and only add a new device, we won't break old code (yet) and we can delete the old code at summer down.

Reply:
It looks like there is no good time to test the new GUI due to dream laser work daily. Myself will be on vocation. I will hold this PR until summer down. Thanks Zach and Robert!